src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './graphqlConfig';
 export * from './credentials';
 export * from './user';
+export * from './authenticationHelper';
```

Comment:
Why is this needed? The authentication helper is internal and shouldn't be used by people directly. Is it to solve this issue: https://github.com/realm/realm-graphql/issues/10?